src/webvr-polyfill.js

```diff
@@ -107,7 +107,7 @@ WebVRPolyfill.prototype.enablePolyfill = function() {
   navigator.getVRDisplays = this.getVRDisplays.bind(this);
 
   // Polyfill native VRDisplay.getFrameData
-  if (this.nativeWebVRAvailable && window.VRFrameData) {
+  if (this.nativeWebVRAvailable && this.isCardboardCompatible() && window.VRFrameData) {
```

Comment:
@dmarcos: @ngokevin: see the discussion in https://github.com/googlevr/webvr-polyfill/issues/177#issuecomment-282671833

@brianchirls: do you have thoughts now on a better way to feature-detect for this? or perhaps expose this ["polyfilled" `VRFrameData`](https://github.com/googlevr/webvr-polyfill/pull/264/files#diff-7bfb91b61bb80cff41aba297525c4dc7R144) under a different name?